pkg/templates/kubernetes/resources/target_group_binding.yaml

```diff
@@ -14,8 +14,8 @@ sanitize_name:
 properties:
   Cluster:
     type: resource
-    namespace: true
```

Comment:
Is the binding not namespaced to the cluster?

Reply:
it is, not sure why that got removed